go.mod

```diff
@@ -1,6 +1,8 @@
 module terraform-provider-genesyscloud
 
-go 1.20
+go 1.21
```

Comment:
Just a note for everyone, my goland is automatically doing this as well, I think if you have a higher version of go on your machine, goland will automatically update the go version for the provider, just something to look out for in case you accidentally update it